🤳🏻 Enable collectibles and NFT as profile picture

shivekkhurana opened the pull request:
fixes #12613

### Summary
- Enable NFTs view (based on selected network, only works with mainnet and rinkyby)
- Allow setting NFTs as Profile picture
- View NFT details

- TODO in next PR: Make NFT txns

#### Platforms

- Android
- iOS

#### Areas that maybe impacted
- Profile pictures

##### Functional

- 1-1 chats
- public chats
- group chats
- wallet / transactions
- new account
- user profile updates


status: Ready

shivekkhurana committed:
NFT UI 🖼

New UI and re-frame handlers for setting pfp from url
Add fake chain-id, connect the pfp upload backend to UI
Add toast on setting pfp, new-new-new UI
Show assets based on selected network
Add horzontal padding to traits card
a11y ids and pr review changes
Added emoji in toast, fix chain id nill issue
Restore Podfile
Fix fix podfile
Add placeholder images for collections, hide option to set pfp when image url is empty or ends in mp4 or svg
Improvise missing designs
Fix paddings
New nu placeholders
Sort collections by name
Kinda fix issue 4
Fix lint

Signed-off-by: Shivek Khurana <shivek@status.im>

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_multiplication_x: | 1b1edc0c | [#1](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-12615/1/) | 2021-09-22 10:06:48 | ~16 min | `android` | [:package:`apk`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-210922-095021-1b1edc-pr12615-universal.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-210922-095021-1b1edc-pr12615-universal.apk)|

shivekkhurana commented:
<details>
  <summary>📸 View Screenshot</summary>
  
![Simulator Screen Shot - iPhone 11 - 2021-09-27 at 16 40 52](https://user-images.githubusercontent.com/1925158/134903210-2a3fcecd-30d5-4b7e-8033-c5029bceafd5.png)
</details>

hesterbruikman commented:
@shivekkhurana Could you please make a minor edit to the Collectibles empty state:
It currently says "you will share your **wallet** and IP address", this should be "You will share your **account** and IP address". For consistency. Also, this is my bad, I proposed the copy using 'wallet' earlier.

hesterbruikman commented:
Also curious about the toast message. What flexibility do we have there when it comes to formatting? e.g. Are emoji's an option?

shivekkhurana commented:
> Also curious about the toast message. What flexibility do we have there when it comes to formatting? e.g. Are emoji's an option?

Everything is allowed. All React components are valid toasts, however I cannot promise anything until I check. If you want emoji's send me a design (or a rough description) and I'll try it out.

hesterbruikman commented:
> 
> 
> > Also curious about the toast message. What flexibility do we have there when it comes to formatting? e.g. Are emoji's an option?
> 
> Everything is allowed. All React components are valid toasts, however I cannot promise anything until I check. If you want emoji's send me a design (or a rough description) and I'll try it out.

Awesome! I wasn't thinking of styling the component tbh, but thinking it's a good opportunity for more casual language. We don't use that often, because it's easy to go wrong. But this is a fun low-risk context:

"Profile picture updated😎"

flexsurfer commented on src/status_im/ui/components/toastable_highlight.cljs:
```diff
@@ -0,0 +1,116 @@
+(ns status-im.ui.components.toastable-highlight
+  (:require [reagent.core :as reagent]
```
could you please add a comment what is this component about , thanks

flexsurfer commented on src/status_im/ui/screens/wallet/account/views.cljs:
```diff
@@ -115,13 +117,95 @@
                            :border-color     colors/gray-lighter
                            :border-radius    16}}]])])
 
+(defn nft-trait-card [trait]
```
could we move nft related to separate ns ?

status-im-auto commented:
## 97% of end-end tests have passed
```
Total executed tests: 67
Failed tests: 2
Passed tests: 65
```
```
IDs of failed tests: 6645,6237 
```
<h3>Failed tests (2)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/end-to-end-tests/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=develop&APK_NAME=12615.apk&PR_ID=12615&TR_CASE_IDS=6645,6237">Rerun tests</a></li><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319636">test_restore_account_migrate_multiaccount_to_keycard</a>, id: 6645 </b></td></tr><tr><td><p><blockquote><div>Device 1: *Tap on found Button* </div><div>Device 1: *Find Button by accessibility id:* `back-button` </div></blockquote></p><code> Device 1: Button by accessibility id: `back-button` is not found on the screen</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c2d3f51d941c4f7392a9c3618f477162?auth=7dc8091c1ed3f2e3fbc623659792d25b">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/c2d3f51d941c4f7392a9c3618f477162/0130screenshot.png?auth=7dc8091c1ed3f2e3fbc623659792d25b">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319616">test_fetching_balance_after_offline</a>, id: 6237 </b></td></tr><tr><td><p><blockquote><div>Device 1: **Toggling airplane mode** </div><div>Device 1: *Find AirplaneModeButton by accessibility id:* `Airplane mode` </div></blockquote></p><code> Device 1: AirplaneModeButton by accessibility id: `Airplane mode` is not found on the screen</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/7934fca504894b3ca7d3676a2ac8c76e?auth=fd61f3c66a478cbb3b4dc6b408023065">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/7934fca504894b3ca7d3676a2ac8c76e/0026screenshot.png?auth=fd61f3c66a478cbb3b4dc6b408023065">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (65)</h3><details><summary>Click to expand</summary><br/><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319631">test_image_in_one_to_one_send_save_reply_timeline</a>, id: 6305 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/05fdc0c095534eefa7dd50e2131077c8?auth=858c8ef89a28755af4d6e908ada8fb7e">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/8327f20e5f944861a8ec83fd07b11fd0?auth=8a545cb921e0b8f6fcfdccab5f96cced">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319613">test_add_account_to_multiaccount_instance_generate_new</a>, id: 6224 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f66230f761a244859fbf6f472f60c542?auth=44689a8ebb0f44e802b233dbd2c6070c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319602">test_can_add_existing_ens_on_mainnet</a>, id: 5502 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5dff2e722bdc4d26978e39a753b66189?auth=ecb48a35233eabc49a08e86268ba87c1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319628">test_keycard_can_see_all_transactions_in_history</a>, id: 6291 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/99d62cbba3204dbdb02b055357087830?auth=db1dd046ece7be72e1f7ab382aabedc5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319586">test_sign_message_and_2tx_in_batch_and_transactions_filters_from_daap</a>, id: 5342 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b5fb98c5e9da4a3bb7f8650ff2cb6567?auth=a4d7765e41d1fe417e3d08ad95134760">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319618">test_add_and_delete_watch_only_account_to_multiaccount_instance</a>, id: 6244 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/def25b83ba254332b9914fa2aeae20e3?auth=1d0d089afad6dba9f1b7afad0f9e80c1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319612">test_open_blocked_secure_not_secure_inlalid_offline_urls</a>, id: 6210 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/52bcb4cafe854f44950b03807faa231d?auth=b02acc0db130521b0853c1b068800177">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319598">test_open_public_chat_using_deep_link</a>, id: 5396 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2c37e625bf6e42548ff258d43d77a9db?auth=14e9c48102317dae1d1f105bc802ac33">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319572">test_offline_add_new_group_chat_member</a>, id: 3998 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b7d0f9f2cc8340f7872359a20f27ec62?auth=8880bbf36aaddf9bf58d87f432144ded">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/02ed2b43471140f5bb744948d0ab8855?auth=11e5f9fe90516dd0ab4f5a37f3f801c4">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://saucelabs.com/jobs/afd572b7772349dcbb892680946d3653?auth=af828681e7f97484be562589267fde18">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319579">test_send_non_english_message_to_newly_added_contact_on_different_networks</a>, id: 5315 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/4e18d4e915044e2694469ca7cadfcfbe?auth=f36adef99d5839bdb34c6b2f20bee80c">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/8b7b0b6e8d284968a1077b270563eeb8?auth=1668abd2e9d13095769bef1f6b8a97b3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319578">test_can_see_balance_and_all_transactions_history_on_cellular</a>, id: 5314 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/67020588a92947a69bba669228d11017?auth=ac83ada26ceb9362e3a19e93f5c92bd9">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>12. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319600">test_insufficient_funds_wallet_positive_balance</a>, id: 5412 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8e5b73b3ea7241e89852b328b09fee12?auth=e5d32189d059239996c0be4853f6e269">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>13. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319596">test_delete_close_all_tabs</a>, id: 5390 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d5c59a4090ba4e789e31de24a3bf3b23?auth=ba33bc784ddc303c27ecf3329de23cad">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>14. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319571">test_create_new_group_chat_messaging_pn_delivered</a>, id: 3994 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5a64d16ba1d94bf8a53be596fa644967?auth=5d9a77fb180f24490786641022f29026">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/58ad0a60715b40f5a085aae7e4375da1?auth=52a54aa7b0d58777919e09cbaa3f8d4b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>15. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319588">test_send_token_with_7_decimals</a>, id: 5350 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/884c6f2a879241e0a5bfb246273a4717?auth=ce2f5569b627c6e136bdf4966ae4fd48">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>16. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319595">test_delete_chats_via_delete_button_rejoin</a>, id: 5387 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5e60637ddbaa42aca83ec4b423c1b0c8?auth=4617404dbe1226447e81a64d9324e2f7">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>17. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319634">test_send_transaction_set_recipient_options</a>, id: 6328 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9e92acab3f57499e8021a15be1d9313e?auth=98f1771ba1c0eac6f12ed8721216b665">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>18. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319620">test_keycard_sign_message_and_transactions_from_daap</a>, id: 6251 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d3b10dd273fd4b0c834f80b0768be9e8?auth=41ef4965f721e0616d884f45fc04c248">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>19. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319630">test_recover_account_from_new_user_seedphrase</a>, id: 6296 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/10bea7bd152b4af9bbbaf86b9e35dc4a?auth=bf9e6cd413e28eab34b206b866828ca1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>20. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319606">test_keycard_can_recover_keycard_account_card_pairing</a>, id: 5758 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c1610a778f1a4424bffa316b0e2c873e?auth=3a6c0c59cf02837d6365a6b99a46603a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>21. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319610">test_block_user_from_public_chat</a>, id: 5786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6b7cf55bb8444a429ac473434af50424?auth=9bb6b087531f6aa141d3c5975d491675">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/64b6addb166340d5895def101bf61711?auth=c01d448b43d46109da377f4ccfe3cfc5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>22. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319633">test_send_audio_message_with_push_notification_check</a>, id: 6316 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/51fa0d53e4924de0a2fe309b40158c52?auth=6c9f0c97be7c5299b4b67f48c307c6ea">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/4aea82a297ee47609a51b030a4aeccef?auth=8dfb3a8417ff8d82816586c17c28aea1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>23. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319603">test_redirect_to_public_chat_tapping_tag_message_fetch_more_history</a>, id: 5675 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0c5bfde7e5894692bd83205ecb23bbf9?auth=67f5e7d43e0a7e14f9d03e505a374f55">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>24. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319638">test_edit_delete_message_in_one_to_one_and_public_chats</a>, id: 695843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f8cbb7af995b48e18ab64086338ccf23?auth=747a16d08bdf13fcf5d5fdb411e517ba">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/685a3f285496476ca85dd028b7792847?auth=fac78a4178cc270bfcd1419b30b65f0b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>25. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319614">test_ens_mentions_pn_and_nickname_in_public_and_1_1_chats</a>, id: 6226 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/44b505e5fe5b4407a05b09bfdfcd38f8?auth=12f31c2321ed7488e68dc54789007e5f">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/b11638cc9c43472bacd897f326d4b63b?auth=7f707c04c8faada8a80ca667f7b08ecd">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>26. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319627">test_keycard_send_eth_from_wallet_to_address</a>, id: 6289 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d0572a7a76d74d7da6a537bb0a123d99?auth=28cc6a581a56413cb0398be50c0e695a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>27. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319604">test_keycard_create_login_restore_unlock_same_seed</a>, id: 5689 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e7f62fc809a54b61941d171a7c32d758?auth=be9b1da492e49d6fd5aedd000e6517b5">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/730e47f87c674960929c261df2bfe6b8?auth=81ceeccd4a483e661bb385b820b22b32">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>28. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319624">test_add_account_to_wallet_private_key_and_seed_phrase</a>, id: 6272 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3210ea012a304af7b06ad4b8016e7cb0?auth=18bd67f1cfd819de50ea791e3c738ee0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>29. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319576">test_offline_is_shown_messaging_1_1_chat_sent_delivered</a>, id: 5310 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f2f383fc1cd44616bf8505244f056c6d?auth=dfceea70ccfdd503b6825d4c41040fb9">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/f7dee7c5a5e042b6a39d8f7a0b2a287f?auth=16a8e5792b6ed23b67d99a84e9f41d9a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>30. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319607">test_pair_devices_sync_one_to_one_contacts_nicknames_public_chat</a>, id: 5762 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ccbd490dbee94bb7be045cf76544b08a?auth=d674d83a58d776aa527121a5e0bd01a5">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/83d2e81de41d44ffb47c6a65c5a954d6?auth=9e0115238c9473eaa7af55acc21ec5d8">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://saucelabs.com/jobs/756d9942ab084e08ab5579da6ece1a05?auth=c0a6ddfd9949fa468f19b71b54538641">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>31. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319581">test_long_press_to_delete_chat</a>, id: 5319 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/660ebe4507204438a0b0f2675782f3c2?auth=e27294e3020fac67f868d6b247008413">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>32. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319580">test_copy_and_paste_messages</a>, id: 5317 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5b2ef7eed41046ca9d56cdad1f48ddf6?auth=d96499eea9382ea23237a78660e1d413">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>33. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319594">test_open_transaction_on_etherscan_copy_tx_hash</a>, id: 5384 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0d572e4aacb449e4bd718ddca4c2b820?auth=6bf7b73286c10a05e7a9c7586dd603cf">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>34. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319590">test_unread_messages_counter_public_chat</a>, id: 5360 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/55594372a6dc4e2985f0618b74a108c0?auth=8b2142eea1705d601e33f9dd85460352">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/3a194621adf04f3194c7fb1977adcaff?auth=9ca77596766cf3d1f3e4d336bada7a4c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>35. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319574">test_send_eth_from_wallet_to_address_incorrect_password</a>, id: 5308 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a4bce9eb62694228b59e1d7e3f951144?auth=5b9504f29d10dbc9d6b07437b3c02827">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>36. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319615">test_mobile_data_usage_complex_settings</a>, id: 6228 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/38efd1c0436f409b82bb090f20fd1d5e?auth=ae4a2cbab7a56ef9c2d1fa51035ec193">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/cd226e7504894df5bdabb7398a630b66?auth=be4cca65307911cb85e0a64a64206631">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>37. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319599">test_start_chat_with_ens_mention_in_one_to_one</a>, id: 5403 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/fee9825a9ff94cfc8a6e98d5192e135c?auth=ed8db6475bca6bf8ad14804461220178">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>38. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319623">test_decline_transactions_in_1_1_chat_push_notification_changing_state</a>, id: 6265 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b50a7580cc0c442fb86e5f8b00e14402?auth=fbd831423f28613162cff1ceaebc8aff">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/39605311c3ed4ec6af78fb52fd95f724?auth=733f0730dec12bf1444e1b597e10fe1a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>39. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319621">test_send_eth_in_1_1_chat_transaction_push</a>, id: 6253 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/df65925d7cda436cbbbde09630b28779?auth=8ec8818666634fbaea5d7104b79ae55c">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/d29e35f45f8946df8634af11ae6c680b?auth=4d5e874565cf8652e539f6b40095853e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>40. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319608">test_install_pack_and_send_sticker</a>, id: 5782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bf2b0eb3bde74f938236e7ad51bbec5d?auth=2911e53161560a88761f7578aa42270d">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/4858b1d9aa9249259335e442d2d161f2?auth=e9b43e7939eba2544c310fc6dcb83b2c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>41. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319589">test_switch_users_special_char_password_and_add_new_account_logcat</a>, id: 5356 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/29a9af90516a46e19b3e49588d62fc4e?auth=2c93f2dc6eb364a3ba7d080c36e31c05">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>42. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319585">test_manage_assets</a>, id: 5341 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1620345375004fcf99672e2961d4a4d1?auth=83fa04dabdcf18ed2d9f6c7ac7b4404e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>43. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319635">test_browser_managing_bookmarks</a>, id: 6633 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f3324faad7fb44f996feca1d6dcb3c68?auth=05e22a293fdc810899d0e1d62938d7ed">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>44. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319573">test_open_chat_by_pasting_chat_key_check_invalid_chat_key_cases</a>, id: 5304 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/aa7106cdd5fa443f9c06cc3a90c755ea?auth=4ac6e98d3e56d3ca491a8adfdf42ee05">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>45. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319575">test_request_stt_from_daap</a>, id: 5309 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/444d645fb2e34f6eb0de39b497e2cf59?auth=956f4f2eba6136d1f667f7441364aafe">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>46. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319609">test_can_use_purchased_stickers_on_recovered_account</a>, id: 5783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a786c283d06a45179e3c966b5051ca48?auth=980236d00ee2d804dd5cdfdadef2762b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>47. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319605">test_dapps_permissions</a>, id: 5738 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/74065ce236814b9ea2376b8ec16aa262?auth=59d3a005c4d8c2e7bc7987ec9ee92f5a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>48. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319597">test_account_recovery_with_uppercase_whitespaces_seed_phrase_special_char_passw_logcat</a>, id: 5394 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c39e9369a9f64404b8f74d42bdce61a7?auth=f148c7ddb0de06f84dcf3fd7ea0393bb">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>49. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319625">test_send_eth_to_ens_in_chat</a>, id: 6279 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ab40a00f362845658165022284903ff8?auth=59beb42bfc52e37704653a3edcfed97c">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/29259d1cad63424ab905aaabb9e940ec?auth=815d650f228c3d03c5ab9dbb520d755c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>50. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319582">test_share_copy_contact_code_and_wallet_address</a>, id: 5323 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9cb79db2a6934f819e5d7da5eaf0fc14?auth=fdd9411298bb805901c53ac4bcf73458">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>51. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319584">test_wallet_set_up</a>, id: 5335 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8b8b1a52da65428bb76c49e39de9837c?auth=7ec7acb3ecd6b108241aa3b6bb00361d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>52. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319611">test_send_transaction_with_custom_token</a>, id: 6208 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/94658ab113bb446ebf6f93fa7bf98e10?auth=0c82ce1d89d4bcb76379190cb5b235f0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>53. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319593">test_home_view</a>, id: 5379 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/74eab9a223ae497988dfdbd26b4405fa?auth=3ee401d8a5d27d28cb9b1019bb9dc22a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>54. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319601">test_logcat_backup_recovery_phrase</a>, id: 5419 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b69303ed49514b49ad9957636d1d1750?auth=3bfa1262d2c96f7e8eb60dd25d6e542d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>55. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319622">test_request_and_receive_stt_in_1_1_chat_offline</a>, id: 6263 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/7b0d4169af89469d9343216281f4d3fb?auth=7a1a3f7d94a6e270ffcf6214d6c8a658">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/fe5370bae0ab4d759579a0f123b8cbbb?auth=0660f188bb97c3d18192ce7730b28ddf">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>56. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319637">test_set_profile_picture</a>, id: 6646 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5f9007a2fc5540bbb7ed33a10cc7c362?auth=b8100ee394a418569cd75af0a002e8e4">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/971c3deb53aa4ae7938de0d5d58b5a9a?auth=f2501de8d08c4778c7f7216d7b041bd3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>57. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319583">test_add_and_remove_mention_contact_with_nickname_from_public_chat</a>, id: 5332 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9a730f4efd4f4250bacbe346fdeb9c16?auth=ef0405fef2ff05817146fdec3a83848d">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/754f241fc2c54319aac5627cf7e7bbc1?auth=031cc460c459bf902518ddefa52176dc">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>58. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319619">test_keycard_request_stt_from_daap</a>, id: 6249 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/7c68a437720e41cdbc2cf62b9cfe3f31?auth=da668c8baba4a632ef978bb09e3a091a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>59. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319617">test_restore_account_from_mnemonic_to_keycard</a>, id: 6240 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c347acf18ca547adaece810caffb17cb?auth=bf2988ff552b576a729b17255611d9f1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>60. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319591">test_pass_phrase_validation</a>, id: 5363 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/48dc39defdd54907acea153756caa209?auth=99cb2a62ab643efc7caa1e66867d557a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>61. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319629">test_keycard_send_eth_in_1_1_chat</a>, id: 6293 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d57142e3b53642b095522aa4aab31b82?auth=95eea4ea05feb42f2b94b5cacaab2f07">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/0b92037a4cf14103b1cbf501665283aa?auth=117448cdbac6acd218ef3ea9bd7efb58">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>62. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319592">test_send_and_open_links_with_previews</a>, id: 5373 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/7225f5f9ea044f0898143cb1d703610f?auth=54f3388264eb5f64b92885bd2330703b">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/d941099820024d21b0ac06e1e0171936?auth=23cca0e47d430dff04da5eb5167fe031">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>63. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319632">test_reactions_to_message_in_chats</a>, id: 6315 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/685cc981c3504e5b99d2bc55d47b8fa2?auth=3ebcfcab1eab4add4e713cee2131747e">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/6d25335ba74e48d7ba732edc10eaf0f2?auth=dc8d6547e5937bbb58fec95555f6ec6f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>64. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319626">test_push_notification_1_1_chat_no_pn_activity_center</a>, id: 6283 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ea519eba122145dbad5d44dadbd4b883?auth=668bc84db89a9101b1e88545ddaa4550">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/a08130916e394c4da6150d243ef40096?auth=83f8121c2d6414ba7142cf5049d7811f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>65. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2319577">test_public_chat_messaging_emojis_timestamps</a>, id: 5313 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b2411e25464b4fa681cb56e61ace83d6?auth=39f85e4103d82b6a83541ae36bd4c8d6">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/9de287134f094f94b6f80932589fb879?auth=c33a72a1323fc269e91193a7e168c617">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

status-im-auto commented:
## 100% of end-end tests have passed
```
Total executed tests: 2
Failed tests: 0
Passed tests: 2
```
<h3>Passed tests (2)</h3><details><summary>Click to expand</summary><br/><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2320209">test_restore_account_migrate_multiaccount_to_keycard</a>, id: 6645 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/138bab0fde8d4775add12957af15a7f7?auth=1700fc3e3211465dcbcfee636564d398">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2320208">test_fetching_balance_after_offline</a>, id: 6237 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/058fdee5bfaf4a0592683a43562854a7?auth=dba49f8b2da88ced26143c7b21573cf0">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

churik commented:
@shivekkhurana 
I can't see any images of collectibles on my Android 10.

![IMAGE 2021-09-29 14:23:04](https://user-images.githubusercontent.com/4557972/135267552-8f16941b-14c7-4859-9b36-e22153d3ce0e.jpg)

IOS build can't be built.
Can you please take a look?

shivekkhurana commented:
@chu Do you have an error for ios builds?

churik commented:
Can be found here: https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-12615/7/console

jakubgs commented:
No idea why but doing `make nix-update-pods` in a separate PR worked and fixed issues in this one: https://github.com/status-im/status-react/pull/12676

churik commented:
### ISSUE 1: Missing translation in settings
![IMAGE 2021-10-01 16:29:07](https://user-images.githubusercontent.com/4557972/135637080-35958ba1-e1a5-4b8b-a038-b615b60a4010.jpg)


### ISSUE 2: Still can see amount of collectibles on Ropsten network, but they are not tappable

**Steps:**
1) restore multiaccount with collectible from seed phrase(PR build by default is using Ropsten)
2) Enable Collectibles

**Expected result:**
Collectibles are disabled, it is explained somehow in UI (i.e. `Collectibles are shown only on mainnet`)

**Actual result:**

![IMAGE 2021-10-01 16:41:35](https://user-images.githubusercontent.com/4557972/135639132-4ae3bdf8-9cc2-4449-acbe-4f7bab0a8913.jpg)
**OS:** IOS, Android

### ISSUE 3: Handling of collectibles without images

- [ ] 3.1. Set some placeholder for collections without images 
<img width="183" alt="Monosnap 2021-10-01 16-52-20" src="https://user-images.githubusercontent.com/4557972/135641141-122e6314-e2ab-4fa9-a011-d4ddbc647ba2.png">

- [ ] 3.2. Set placeholders for collectibles without placeholders
<img width="187" alt="Monosnap 2021-10-01 16-53-31" src="https://user-images.githubusercontent.com/4557972/135641329-032beec4-726e-44d4-a104-621ad6152d64.png">

- [ ] 3.3. Disable `Use as profile photo` for collectibles without images as it can't be set
<img width="185" alt="Monosnap 2021-10-01 16-56-08" src="https://user-images.githubusercontent.com/4557972/135641778-43358aa9-e253-4cac-8b63-777c19158be1.png">

### ISSUE 4: Number of collectibles on OpenSea website and in status app is different

**Steps:**
1) add watch-only account to multiaccount (`0xcf2272205cc0cf96Cfbb9Dd740BD681D1E86901E`)
2) count collectibles
3) tap `Open in Opensea` and compare value

**Expected result:**
amount of collectibles should be the same (12)
**Actual result:**
12 in OpenSea, 13 in Status
The redundant is MegaCryptoPolis, it is not tappable from wallet.
**OS:** IOS, Android

### ISSUE 5 (question): why the background color is black when you set collectible as profile photo? 
![IMAGE 2021-10-01 17:18:46](https://user-images.githubusercontent.com/4557972/135645349-ee4c9e82-be6f-45b7-8e39-76171c89c7ac.jpg)

rasom commented on src/status_im/multiaccounts/core.cljs:
```diff
@@ -198,6 +198,20 @@
+    (fx/merge cofx
+              {::json-rpc/call [{:method     "multiaccounts_storeIdentityImageFromURL"
+                                 :params     [key-uid url]
+                                 :on-error   (partial prn :----->)
```
`log/error`/`log/warn`

rasom commented on src/status_im/ui/screens/chat/styles/photos.cljs:
```diff
@@ -6,8 +6,8 @@
 (defn photo-container [size]
-  {:position      :relative
-   :border-radius (radius size)})
+  {:position         :relative
```
unnecessary change

rasom commented on src/status_im/ui/screens/wallet/collectibles/views.cljs:
```diff
@@ -0,0 +1,223 @@
+                                                                      :border-width  1
+                                                                      :border-color  colors/gray-lighter}
+                                                             :source {:uri (:image_url collectible)}}]
+                         :main-icons/photo)
```
aligning is a bit off

shivekkhurana commented:
## ISSUE4 
Anthony and I did some detective work to find what the issue is.

There are two aspects of the OpenSea api:
1- A list of collections
2- Actual assets list (can get all assets or filter by collection id)

The failing collections (MCP in this case) have 0 assets returned although the collection says 1 asset available. So the app stays stuck at loading state (grayish box with no icon). Rainbow works fine because the don't query collections API and simply fetch all assets (ie. fat payload) and then compute collections from all assets.

We face a problem because we fetch assets on demand, and it gets stuck when there are 0 assets in the collection.

- One possible "easy" solution is to show an error when there are 0 assets in a collection **DONE** ✅
- Other possible solution is to fetch all assets upfront but this will hamper UX because we will be loading all assets upfront. **Dropped - Because fat payloads are no good** 🙅🏻

## ISSUE5 No sorting, so no way to find smth you need in random list
Better to sort them alphabetically **DONE** ✅

---

Also addressed comments from Roman. I have not enabled it in release version yet, but I think its good to go in develop. That will allow a broader review.
